oam/install_scripts/mariadb-columnstore-stop.sh

```diff
@@ -8,7 +8,7 @@
 /bin/systemctl stop mcs-writeengineserver
 /bin/systemctl stop mcs-primproc
 /bin/systemctl stop mcs-controllernode
-/bin/systemctl stop mcs-workernode
+/bin/systemctl stop mcs-workernode@1.service mcs-workenrode@2.service
```

Comment:
I'm curios why do we stop the 2nd instance of the service?

Reply:
I have removed as non-primary workernodes are managed by cmapi